packages/core/src/Table/Table.js

```diff
@@ -20,12 +21,19 @@ const tableStyles = css`
  * @example import { Table } from '@dhis2/ui-core'
  * @see Live demo: {@link /demo/?path=/story/table--static-layout|Storybook}
  */
-export const Table = ({ children, className, dataTest }) => (
-    <table className={className} data-test={dataTest}>
-        {children}
+export const Table = ({
+    children,
+    className,
+    dataTest,
+    suppressZebraStriping,
+}) => (
+    <Provider value={{ suppressZebraStriping }}>
```

Comment:
I was thinking whether it makes sense to set a default value for suppressZebraStriping in the Table's defaultProps, instead of in the context. Instinctively I'd have gone with defaultProps since it feels like it belongs more to the Table to me. What are your thoughts on that @Mohammer5?

Reply:
That's what I did initially when the default value was `true`. But after having changed the prop, I thought it be better to be consistent with all our other components where a missing flag means `false` and we don't "mention" that in the default props.

Reply:
Ok, so the default in the context is not as necessary. All right, yeah I guess there's not really any real harm in having it, other than that there's an implicit default in the Table and an explicit one in the context.